docs/index.md

```diff
@@ -43,6 +57,27 @@ core_lib:
         protocol: sqlite
   ...
 ```
+#### `your_core_lib.yaml` Explained:
+<b>your_core_lib.yaml is the setting for your entire Core-Lib library. The above example will show how to configure core-lib to connect to a database using [SQLAlchemy](https://docs.sqlalchemy.org/en/20/):</b>
+- <b>log_queries:</b>False, meaning that SQL queries won't be logged. True, [echo flag](https://docs.sqlalchemy.org/en/20/core/engines.html#more-on-the-echo-flag) is set to true.
```

Comment:
1. keep the `default: True` formatted as other place in the docs. 
2. do this for all places you mention default parameters. 
3. put the default: true in the begging or end. according to the existing standard. not in the middle of a sentence 


also see if we can configure any link we have to open in new tab [Link Text](URL){:target="_blank"}
as long that this links are pointing to external resource.